src/chrome/content/rules/MERY.jp.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```

Comment:
Ah, indeed, and even then, redirecting to another domain can be completely fine; it's just that it turned out to break a lot of rulesets in the case of Akamai at one point in time. Apologies for any confusion.